### PR TITLE
Allow for modulepaths other than pwd

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

File: generate_answers.rb [hunk 1]
```diff
@@ -79,8 +79,10 @@ def puppetmaster_questions
 # helper methods
 
 def save_or_run_and_exit
+  # If the foreman_installer module exists then just use it. Otherwise, ask.
+  $modulepath = File.exists?("/usr/share/foreman-installer/foreman_installer") ? "/usr/share/foreman-installer" : ask("<%= color('Where are the installer Puppet modules?', :question) %>")
```
Comment (domcleal) [inline generate_answers.rb]: The default might be better as `File.dirname(__FILE__)` so it works from a checkout too.

**Comment by domcleal** [comment]
The $outfile also needs fixing as it's relative, shall I do that separately?

**Comment by skottler** [comment]
I've updated $outfile and made that changes mentioned in the review. Thanks @domcleal!

**Comment by domcleal** [comment]
Merged, thanks!

**Pull request closed without merging**